.dockerignore

```diff
@@ -0,0 +1,3 @@
+./build
```

Comment:
Nit: Should we also add `.git` and `*.log` like in https://github.com/mozilla-services/Dockerflow/blob/93a7f0e0f69fee957d5bfbdac236ab876356cb95/.dockerignore#L1 ?